### PR TITLE
tests: Add docker top integration test.

### LINKED ISSUE #812: Add docker top test
Tests docker top functionality

### PR DESCRIPTION
**Pull request by GabyCT**
This tests the docker top functionality.

Fixes #812

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: integration/docker/top_test.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+var _ = Describe("docker top", func() {
+	var (
+		id		string
+		exitCode	int
+	)
+
+	BeforeEach(func() {
+		id = randomDockerName()
+		_, _, exitCode = DockerRun("--name", id, "-d", Image, "sh", "-c", "sleep 10")
```
Comment (devimc) [inline integration/docker/top_test.go]: ```sh -c``` is not needed

File: integration/docker/top_test.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+	. "github.com/clearcontainers/tests"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("docker top", func() {
+	var (
+		id		string
+		exitCode	int
```
Comment (devimc) [inline integration/docker/top_test.go]: define the workload as a variable

```go
workload string
```

File: integration/docker/top_test.go [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	)
+
+	BeforeEach(func() {
+		id = randomDockerName()
```
Comment (devimc) [inline integration/docker/top_test.go]: initialize the workload

```go
workload = "sleep 10"
```

File: integration/docker/top_test.go (hunk 1 above)
Comment (devimc) [reply integration/docker/top_test.go]: use the workload

File: integration/docker/top_test.go [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+		Expect(RemoveDockerContainer(id)).To(BeTrue())
+		Expect(ExistDockerContainer(id)).NotTo(BeTrue())
+	})
+
+	Context("check docker top functionality", func() {
+		It("should print usage statement", func() {
+			Skip("Issue: https://github.com/clearcontainers/runtime/issues/876")
+			stdout, _, exitCode = DockerTop(id, "-x")
+			Expect(exitCode).To(Equal(containerExitCode))
+			Expect(stdout).To(ContainSubstring("sleep 10"))
```
Comment (devimc) [inline integration/docker/top_test.go]: use the workload

```go
Expect(stdout).To(ContainSubstring(workload))
```

**Comment by GabyCT** [comment]
@devimc changes applied

**Comment by devimc** [comment]
lgtm

**Pull request merged**